packages/runtime/plugin-runtime/src/core/server/stream/afterTemplate.ts

```diff
@@ -103,9 +103,16 @@ function createReplaceSSRData(options: {
   };
   const attrsStr = attributesToString({ nonce });
 
-  const ssrDataScript = `
-    <script${attrsStr}>window._SSR_DATA = ${serializeJson(ssrData)}</script>
-    `;
+  const inlineScript =
+    typeof ssrConfig === 'boolean' ? true : ssrConfig?.inlineScript !== false;
+  const useInlineScript = inlineScript !== false;
```

Comment:
this code is same as `ssrData.ts`, can be extracted into a util function.

Reply:
OK